playbooks/templates/.github/workflows/tft.yml

```diff
@@ -21,8 +21,6 @@ jobs:
     if: |
       github.event.issue.pull_request
       && (contains(github.event.comment.body, '[citest]') || contains(github.event.comment.body, '[citest-all]'))
-      && (contains(fromJson('["OWNER", "MEMBER", "COLLABORATOR", "CONTRIBUTOR"]'), github.event.comment.author_association)
-      || contains('systemroller', github.event.comment.user.login))
```

Comment:
where is the permission check done?

Reply:
I removed permission check, it sometimes doesn't work as expected, and is hard to troubleshoot. The has a concurrency check, which blocks it from being run more than once at a time for a PR.

Reply:
We need to ensure that random people on github cannot trigger CI tests

Reply:
Let's add a task to dump $GITHUB_CONTEXT? there is no sensible info there as far as I can tell, see an example in this workflow under Dump github context:
https://github.com/spetrosi/network/actions/runs/9790784152/job/27033149915

Reply:
Just to help with troubleshooting a bit, I had situations where users with the listed author_association were not able to run workflow.

Reply:
good idea - yes, dump GITHUB_CONTEXT